### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by szepeviktor**
Fix few misspellings.

### CODE REVIEW

File: src/Illuminate/Foundation/Console/DocsCommand.php [hunk 1]
```diff
@@ -49,7 +49,7 @@ class DocsCommand extends Command
      *
      * @var string
      */
-    protected $help = 'If you would like to perform a content search against the documention, you may call: <fg=green>php artisan docs -- </><fg=green;options=bold;>search query here</>';
+    protected $help = 'If you would like to perform a content search against the documentation, you may call: <fg=green>php artisan docs -- </><fg=green;options=bold;>search query here</>';
```
Comment (QWp6t) [inline src/Illuminate/Foundation/Console/DocsCommand.php:52]:
```suggestion
    protected $help = 'If you would like to perform a content search against the documention, you may call: <fg=green>php artisan docs -- </><fg=green;options=bold;>search query here</>';
```

Comment (szepeviktor) [reply src/Illuminate/Foundation/Console/DocsCommand.php]: @QWp6t What is the purpose of saying _documention_?

Comment (szepeviktor) [reply src/Illuminate/Foundation/Console/DocsCommand.php]: I see! It is a vendored file, a copy of Laravel.
BTW It was fixed there: https://github.com/laravel/framework/pull/48513/files#diff-9805af46bb68f8e90ca7aee2a48e3b3d3be0d0920a5f95683489f3ccfa4c7899

**Comment by QWp6t** [comment]
Thanks for helping us out! 🙏

**Comment by szepeviktor** [comment]
Glad to contribute.

**Pull request merged**